### PR TITLE
Implemented QE Review feedback for Getting Started Kafka README.adoc

### PR DESCRIPTION
**Pull request by AedinC**
* In step 4 of 'Setting permissions for a service account in a Kafka instance in OpenShift Streams for Apache Kafka' section, amended 'set all resource identifiers and values to Is *. to 'set all resource identifiers to `Is` and all identifier values to `*`.
* In last part of 'Setting permissions for a service account in a Kafka instance in OpenShift Streams for Apache Kafka' section, clarified the admonition note block for add permission. 
* In Step 7 of 'Creating a service account to connect to a Kafka instance in OpenShift Streams for Apache Kafka', removed reference to kafkacat.
* In Additional resources, amended Configuration link so that it points to to our own topics configs doc instead of Kafka configuration details.

### CODE REVIEW

File: docs/kafka/getting-started-kafka/README.adoc [hunk 1]
```diff
@@ -204,7 +204,7 @@ After you create a service account to connect to a Kafka instance, you must also
 . In the *Kafka Instances* page of the web console, click the name of the Kafka instance that you want the service account to access.
 . Click the *Access* tab to view the current ACL for this instance.
 . Click *Manage access*, use the *Account* drop-down menu to select the service account that you previously created, and click *Next*.
-. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and set all resource identifiers and values to `Is *`.
+. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and set all resource identifiers and values to `Is` and all identifier values to `*`.
```
Comment (sterobin) [inline docs/kafka/getting-started-kafka/README.adoc:207]:
```suggestion
. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and set all resource identifiers to `Is` and all identifier values to `*`.
```

File: docs/kafka/getting-started-kafka/README.adoc [hunk 2]
```diff
@@ -300,7 +300,7 @@ endif::[]
 
 [role="_additional-resources"]
 == Additional resources
-* https://kafka.apache.org/081/documentation.html#configuration[Configuration^] in Kafka
+* https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/7d28aec8-e146-44db-a4a5-fafc1f426ca5[Configuration^] in Kafka
```
Comment (sterobin) [inline docs/kafka/getting-started-kafka/README.adoc:303]:
```suggestion
* https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/7d28aec8-e146-44db-a4a5-fafc1f426ca5[_Configuring topics in {product-long-kafka}_^]
```

**Review by sterobin — COMMENTED** [review]
@AedinC Some corrections, but otherwise good.

File: docs/kafka/getting-started-kafka/README.adoc (hunk 2 above)
Comment (AedinC) [reply docs/kafka/getting-started-kafka/README.adoc:303]: Thanks @sterobin

**Review by wtrocki — APPROVED** [review]

**Review by sterobin — APPROVED** [review]

**Pull request merged**